e2e/test/scenarios/embedding/embed-resource-downloads.cy.spec.ts

```diff
@@ -216,6 +306,92 @@ describeWithSnowplowEE(
           export_type: "csv",
         });
       });
+
+      describe("with native question parameters", () => {
```

Comment:
we never had this test before, since I changed the code for question download with parameters I needed to add this test as well. It's not part of the bug though.